src/lib.rs

```diff
@@ -87,6 +99,12 @@ impl<E: Env> Ctap<E> {
                 self.hid
                     .process_hid_packet(&mut self.env, packet, now, &mut self.state)
             }
+            #[cfg(feature = "with_vendor_hid")]
+            Transport::VendorHid => {
+                self.vendor_hid
+                    .process_hid_packet(&mut self.env, packet, now, &mut self.state)
+            }
+            #[cfg(not(feature = "with_vendor_hid"))]
             Transport::VendorHid => HidPacketIterator::none(),
```

Comment:
If we protect the code with a feature, shouldn't we also protect the transport enum value? It looks weird to have that enum value if it's not really used.

Reply:
Done